target/linux/lantiq/files-5.4/arch/mips/boot/dts/lantiq/vr9_buffalo_wbmr-300hpd.dts

```diff
@@ -239,6 +181,48 @@
 	};
 };
 
+&gswip_mdio {
+	phy11: ethernet-phy@11 {
+		reg = <0x11>;
+	};
+	phy12: ethernet-phy@12 {
+		reg = <0x12>;
+	};
+	phy13: ethernet-phy@13 {
+		reg = <0x13>;
+	};
+	phy14: ethernet-phy@14 {
+		reg = <0x14>;
+	};
+};
+
+&gswip_ports {
+	port@2 {
+		reg = <2>;
+		label = "lan2";
```

Comment:
Image [1] show that lan ports are described as lan2-lan4 and lan1 can be used as wan.
port2 -> lan3
port3 -> lan4
port4 -> lan1/wan
port5 -> lan2

[1] https://products.buffalo-technology.com/wp-content/uploads/sites/3/wbmr-300hpd-02-768x1920.png

Reply:
> port4 -> lan1/wan

I would keep it simple though for the labels, and stick to lan1; this will have to be used for uci config later ...

Reply:
good catch @olek2 - thank you!

I went with the suggestion from @adrianschmutzler and configured port4 as "lan1" and updated the patch description accordingly